GCTC-NTGC/gc-digital-talent
# Adds the static 404 page to talentsearch

# ❓Purpose

Adds an error page to handle bad requests in both light and dark mode.

# 🏷️ Related issues
- resolves #1904

# 📋 TODO

- [x] Add `react-intl` strings
- [x] Add to the router
- [x] Update SVGs
- [x] Add frontend/apache redirect?

# 🧪 Testing

> **Note**
> Testing dark mode can only be done in storybook

1. Build talentsearch `npm run production --workspace=talentsearch`
2. Navigate to `/404`
3. Confirm it appears and looks like the mockup

# 📷 Screenshot

<img width="1288" alt="Screen Shot 2022-10-12 at 8 24 05 AM" src="https://user-images.githubusercontent.com/4127998/195443470-6e985333-1278-4590-9581-c63cd61f8e4a.png">
